linux/LinuxProcess.c

```diff
@@ -289,6 +291,7 @@ void LinuxProcess_writeField(Process* this, RichString* str, ProcessField field)
          attr |= A_BOLD;
       xSnprintf(buffer, n, "%5lu ", lp->ctxt_diff);
       break;
+   case SECATTR: snprintf(buffer, n, "%-30s   ", lp->secattr ? lp->secattr : "?"); break;
```

Comment:
Use `xSnprintf`/`String_snprintf` as the checked variant?

Reply:
The security attribute can and will be quite long (especially with SELinux).
AppArmor example: `/usr/sbin/cups-browsed (enforce)`

So in case it exceeds the buffer limit, htop should not stop.